lightning/src/routing/scoring.rs

```diff
@@ -391,12 +402,14 @@ pub struct ProbabilisticScoringParameters {
 	/// [`liquidity_offset_half_life`]: Self::liquidity_offset_half_life
 	pub liquidity_penalty_multiplier_msat: u64,
 
-	/// The time required to elapse before any knowledge learned about channel liquidity balances is
-	/// cut in half.
+	/// We track upper and lower bounds on the available liquidity in a channel in order to
+	/// calculate success probability given recent successes and failures. In order to ensure we do
+	/// not get stuck avoiding channels which only failed once, we decay these bounds over time.
```

Comment:
The first sentence should define *what* the parameter is, IMHO. Further paragraphs can provide the larger picture.